### ISSUE TITLE
cat recovery API documentation is not complete

### ISSUE THREAD

**Issue by sylvie777** [elastic/elasticsearch#41986]
Reported by @111andre111

Provide the following information to help us identify and troubleshoot the issue.
- Page URL: https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-recovery.html

There are missing parameters in that documentation which I found here:
https://github.com/elastic/elasticsearch/blob/6.7/docs/reference/cat/recovery.asciidoc

I got a hint from CSH about a flag of the _cat/recovery with the parameter active_only, but you can't find it there.

Here you can find a hint about that a little bit:
https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-recovery.html
But even then there is still one parameter missing.

I had a look into the source code here:
https://github.com/elastic/elasticsearch/blob/6.7/server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestRecoveryAction.java#L54
There you see three parameters:
- active_only
- detailed
- index

But this is even not represented in the 
https://github.com/elastic/elasticsearch/blob/6.7/rest-api-spec/src/main/resources/rest-api-spec/api/indices.recovery.json

So to summarise that, for that customer it is not intuitive to find the correct parameters at least for the recovery API.

By the way a swagger documentation like this would be very helpful and more intuitive I think https://petstore.swagger.io.
That would be cool to have something like this.

Thank you in advance

André
